packages/datetime/src/dateRangeInput.tsx

```diff
@@ -111,36 +113,40 @@ export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {
 
 export interface IDateRangeInputState {
     isOpen?: boolean;
+    preferredBoundaryToModify?: DateRangeBoundary;
```

Comment:
is the `preferred` notion really valuable here or can we just call it `boundaryToModify`?